scripts/dts/extract/globals.py

```diff
@@ -331,6 +331,54 @@ def build_cell_array(prop_array):
 
     return ret_array
 
+def child_to_parent_unmap(cell_parent, gpio_index):
+    # Aim of this function is to return a gpio-controller from the
+    # cell_parent provided (which has been identified as a 'nexus node',
+    # ie: has a 'gpio-map' property) thanks to the gpio_index provided.
+    # Note: Nexus nodes and gpio-map property are described in devide tree
+    # specification, v0.3, chapter 'Nexus nodes and Specifier Mapping'.
```

Comment:
Is "v0.3" a typo? Can't find that version online.

Reply:
It's available here: https://github.com/devicetree-org/devicetree-specification/blob/4b1dac80eaca45b4babf5299452a951008a5d864/source/devicetree-basics.rst#nexus-nodes-and-specifier-mapping.
"Soon to be released" according to maintainer.

Reply:
Sure it'll be called v0.3?

Reply:
Current one is v0.2, so I guess this is a reasonable assumption (https://www.devicetree.org/)

Reply:
Bad idea to refer to a v0.3 version that's not out and not sure to be called v0.3 I think. Could say something like "in the upcoming (presumably v0.3) Device Tree specification" instead.

Small typo: devide -> device